tests/test_riakcs.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from tests.common import load_check, read_data_from_file
+
+
+def _get_stats(self, s3, aggregation_key):
+    return self.load_json(read_data_from_file("riakcs_in.json"))
+
+def _connect(self, instance):
+    return None, None, ["aggregation_key:localhost:8080"]
+
+class RiakCSTest(unittest.TestCase):
```

Comment:
Could you try to use the new test interface for this check too? see `AgentCheckTest` in `tests.common`. Should be easier to assert metrics and tags with that ?
You can copy from the rabbitmq test. Also FWIW I prefer using mock instead of lambdas for tests :smile: